cogs/AI.py

```diff
@@ -13,23 +13,37 @@ def __init__(self, bot: OGIROID):
         self.bot = bot
 
     @commands.slash_command(description="Generates ai art")
-    async def ai_art(self, inter: disnake.ApplicationCommandInteraction, text):
+    async def ai_art(
+        self, inter: disnake.ApplicationCommandInteraction, text: str
+    ):
+        if profanity.contains_profanity(text):
+            return await inter.send(
+                f"NSFW requests are not allowed!", ephemeral=True
+            )
+        if "bot" in inter.channel.name or "command" in inter.channel.name:
```

Comment:
just do if inter.channel.name in ['command'. 'bot']

Reply:
thats true